packages/react-composites/src/composites/common/AvatarPersona.tsx

```diff
@@ -87,3 +82,58 @@ export const AvatarPersona = (props: AvatarPersonaProps): JSX.Element => {
     />
   );
 };
+
+/**
+ * Hook to override the avatar persona data of the users
+ * @private
```

Comment:
We should document what this does a bit better:

* Calls `AvatarPersonaDataCallback` *on each render* for provided `userIds`.
* Returns an array of the same length as `userIds`. Entries in the array may be `undefined` if there is no data to return.

Aside: We currently return an empty list when `dataProvider` is not set. This sort of works in the uses so far, but we really should return an array of the same length as `userIds`, full of `undefined`. This ensures that caller can always zip the input and output arrays.